### ISSUE TITLE
add tests

### ISSUE THREAD

**Issue by X1011** [X1011/git-directory-deploy#6]
write (unit) tests using something like [roundup](https://github.com/bmizerany/roundup/), [Bats](https://github.com/sstephenson/bats), [assert.sh](https://github.com/lehmannro/assert.sh), or [shelltestrunner](http://joyful.com/shelltestrunner/).

write tests for
- [ ] attempting to deploy without setting up first
	- [ ] non-existent or empty deploy_directory (#5)
	- [ ] non-existent local deploy_branch
- [ ] non-existent remote deploy_branch (#5)
- [ ] non-writable remote repo
- [ ] setup should not delete existing files in deploy_directory (i think it does currently if a file also exists in the repo's root)

**Comment by MoOx** [comment]
You just added some right?

**Comment by X1011** [comment]
Yeah, but so far, I've only written a basic happy-case integration test and some unit tests for the code I'm working on for issue #7, so I can't check anything off of this list yet.

**Comment by MoOx** [comment]
No problem. That's awesome. Thanks for you work.
